### DISCUSSION TITLE
Should the litter limit be raised along with the other recently increased limits?

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#16739] [Feature requests]
As mentioned in #16737, there's a limit to how much litter (including vomit) can be in the park. That was presumably a good limit to set it at under the old park sizes...

But now parks can be _much_ larger, but litter still has the hard limit of 500 items in the park, regardless of park size.

Easy and simple solution: Just increase that limit to a new hard limit -- something that makes sense for the larger parks of the NSF.

_But_ that simple solution might result in too much litter being present in smaller parks? So maybe a better solution would be to make the litter limit a function of the overall map size, amount of owned land, and/or amount of path tiles present in the park. A bit more complex to code, but that shouldn't be *too* difficult, right? And it would ensure that there's enough litter possible so that large parks won't feel the lack while also making sure it's limited enough that small parks won't be overwhelmed. (Would probably also need to make sure it can never be below 500, no matter how small the park is.)

**Comment by James103** [comment]
Idea: Guests stop littering when litter count exceeds a value dependent on the number of guests.

In other words, parks with 5,000 guests or less would have a limit of 500 items of litter, but parks with more guests may have more litter at once in order to keep the ratio of the limit being 1 piece of litter per 10 guests (all numbers subject to change).

Note: Any litter over the limit should stay, even when guests leave and the cap lowers as a result.

**Reply by duncanspumpkin** [reply]
The way the limit works is that old litter just disappears to be replaced with new litter when at the limit
